addons/skin.estuary/xml/VideoOSD.xml

```diff
@@ -51,7 +51,7 @@
 							<param name="texture" value="osd/fullscreen/buttons/previous.png"/>
 						</include>
 						<onclick>PlayerControl(Previous)</onclick>
-						<visible>Player.ChapterCount | Integer.IsGreater(Playlist.Length(video),1) | [Player.SeekEnabled + VideoPlayer.Content(livetv)]</visible>
+						<visible>Player.HasSceneMarkers | Player.ChapterCount | Integer.IsGreater(Playlist.Length(video),1) | [Player.SeekEnabled + VideoPlayer.Content(livetv)]</visible>
```

Comment:
May I asked why we need "HasSceneMarkers", but no "HasCuts", "HasEditList"?

Reply:
Because we need a way to control the visibility of other controls that depend on this info (in this case the "Previous" and "Next" items in estuary). For cuts and editlist we currently do not need them as they only affect the range control itself (so if empty no info is shown). This could also be done using only the info interface only on the skin with `!String.IsEmpty(Player.SceneMarkers)`.

I can add HasCuts and HasEdits for the sake of completeness but won't be used anywhere. Can also remove HasSceneMarkers with the alternative above. Let me know.

Reply:
Thanks for the clarification. 
No need to add HasCuts and HasEdits imo as long as there is no actual demand for this.

Reply:
Thanks. Having `Player.HasSceneMarkers` is a bit more performant (since it's core core) than the comparison using infos (`String.IsEmpty` would be anyway which likely would require interpretation).